master/master.cfg

```diff
@@ -326,14 +326,11 @@ class BuilderType:
                 and self.os in ['windows', 'linux'])
 
     def handles_vulkan(self):
-        # Disabled for now, TODO
-        return False
-
         # Stick with Linux on x86-64 for now. Others TBD.
-        # return (self.arch == 'x86'
-        #         and self.bits == 64
-        #         and self.os == 'linux'
-        #         and self.halide_branch not in [HALIDE_RELEASE_14, HALIDE_RELEASE_15])
+        return (self.arch == 'x86'
+                and self.bits == 64
+                and self.os == 'linux'
+                and self.halide_branch not in [HALIDE_RELEASE_14, HALIDE_RELEASE_15])
```

Comment:
These are old releases. Don't we want to test at top-of-tree Halide?

Reply:
Ah, yes! Thanks! I wasn’t aware of the ‘HALIDE_MAIN’ so I was trying to filter out the older branches that I knew didn’t have Vulkan support.

Reply:
Apparently ‘not in’ is a dedicated operator in python. So the preferred path for negation is ‘x not in y’ rather than ‘not x in y’ since that will be implicitly converted .

Reply:
I’m not sure about changing this to just test for ‘HALIDE_MAIN’ … won’t that mean the tests won’t run from the test branches for PRs?

Reply:
No, that means "test everything that is in the main Halide branch"